lib/app/pages/splash.dart

```diff
@@ -11,11 +11,16 @@ import 'package:talao/app/shared/ui/ui.dart';
 import 'package:talao/app/shared/widget/base/page.dart';
 import 'package:talao/app/shared/widget/brand.dart';
 import 'package:talao/deep_link/deep_link.dart';
+import 'package:talao/theme/theme.dart';
```

Comment:
[{
	"resource": "/home/rbarbe/Documents/flutter/spruce/spruceid/credible/lib/app/pages/splash.dart",
	"owner": "_generated_diagnostic_collection_name_#0",
	"code": {
		"value": "unused_import",
		"target": {
			"$mid": 1,
			"external": "https://dart.dev/diagnostics/unused_import",
			"path": "/diagnostics/unused_import",
			"scheme": "https",
			"authority": "dart.dev"
		}
	},
	"severity": 2,
	"message": "Unused import: 'package:talao/app/shared/ui/ui.dart'.\nTry removing the import directive.",
	"source": "dart",
	"startLineNumber": 10,
	"startColumn": 8,
	"endLineNumber": 10,
	"endColumn": 45
}]